dist/images/kubectl-ko

```diff
@@ -26,8 +26,14 @@ showHelp(){
   echo "  env-check    check the environment configuration"
   echo "  tuning {install-fastpath|local-install-fastpath|remove-fastpath|install-stt|local-install-stt|remove-stt} {centos7|centos8}} [kernel-devel-version]    deploy kernel optimisation components to the system"
   echo "  reload    restart all kube-ovn components"
+  echo "  log {kube-ovn|ovn|ovs|linux} {all|sub_component...} [tail_lines]
```

Comment:
Need a short hand to collect logs for all components

Reply:
fixed